.github/workflows/integration_tests.yml

```
@@ -0,0 +1,154 @@
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Run integration tests in client container
+        if: needs.detect-changes.outputs.borg == 'true'
+        timeout-minutes: 5
+        run: |
+          echo "Running integration tests in client container..."
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)